android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -2981,6 +2984,7 @@ private Intent createPingIntent() {
             //the developer can use this pendingIntent to start their SdlService from the context of
             //the active RouterService
             Intent pending = new Intent();
+            pending.setPackage(SDL_PACKAGE);
```

Comment:
See previous comment.

```suggestion
            pending.setPackage(getPackageName());
```